src/api/company.js

```diff
@@ -0,0 +1,12 @@
+import API from './axios'
+
+export const createCompany = async (data, jwt) => {
+  try {
+    API.defaults.headers.common['Authorization'] = `Bearer ${jwt}`
```

Comment:
This shoud go in an interceptor that should be set up in the http service - example implementation: https://jasonwatmore.com/post/2021/09/25/react-axios-interceptor-to-set-auth-header-for-api-requests-if-user-logged-in

Reply:
https://github.com/voi99/uteam-mne/blob/UTM-8/src/api/axios.js#L13 here I set the authorization header, but I'm looking for a token in the local storage and at this time there is no token. So should I add here https://github.com/voi99/uteam-mne/blob/UTM-8/src/pages/JoinPage.js#L28 localStorage.setItem('user',JSON.stringify({jwt:user.jwt}), this will fix the issue but also will be repetitive because at the end of the register logic I am calling login which have similiar functionalities (writing user in the local storage).